### PR TITLE
reshaping the content (+readme.md update)

### PR DESCRIPTION
**Pull request by DT-DawidWroblewski**
fine tuning the content of repos

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
Thanks
The link DetailsPage send me a 404 but I guess this is normal as we did not have yet the 2 API documentation. Correct?

**Comment by DT-DawidWroblewski** [comment]
404 is because I changed the path of md files - I'll update it immediately!

File: documentation/API_documentation/README.MD [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+|---|---|
+|CAMARA|[DetailsPage](/NumberVerification/code/API_definitions/CAMARA/NUMBER_VERIFICATION_API.md)|
+|Mobile Connect|[DetailsPage](/NumberVerification/code/API_definitions/MobileConnect/MC_VERIFIED_MSISDN.md)|
+
+### Endpoint definitions
+
+|Flavour|Endpoint type|Endpoint path|
+|---|---|---|
+|CAMARA|AuthN/AuthZ||
+||resource|/verify|
```
Comment (monamok) [inline documentation/API_documentation/README.MD:63]: /device-phone-number is missing here.

**Review by monamok — COMMENTED** [review]
Sorry I'm commenting late. I was away during last week. I just put my comments here in order to be more clear and for the changes we can open a new PR. 
The idea of having the README file is great but I think with the way the files were moved, we're not aligned with other CAMARA projects anymore. As my understanding, we have to include them like this:
Documentation files --> repository/documentation/xxx.md files (we can still have two subfolders like you did in repository/code/API_definitions). So I think these files (xxx.md files):
https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/CAMARA/NUMBER_VERIFICATION_API.md
https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/MobileConnect/MC_VERIFIED_MSISDN.md

should be moved back to documentation folder, including or not another subfolder.

I also added some other comments about the resources and versioning.

**Pull request merged**